sensu/plugins/metrics-net.rb

```diff
@@ -80,7 +85,7 @@ def run
     Dir.glob('/sys/class/net/*').each do |iface_path|
       next if File.file?(iface_path)
       iface = File.basename(iface_path)
-      next if iface == 'lo'
+      next if !config[:all] and !iface.include?('eth') and !iface.include?('bond')
```

Comment:
I think since the intent is to filter eth* and bond* and not *eth* and *bond*, start_with would be more appropriate than include.